theme/templates/email/signup_verify.txt

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+{% block main %}
+{% trans "Welcome to HydroShare. This email address was used to request an account on HydroShare." %}
+{% trans "If you originated the request, please use the link below to verify your email address and activate your account." %}
+http://{{ request.get_host }}{{ verify_url }}
```

Comment:
@ironfroggy @mjstealey @selimnairb I tried to change http to https, but it will not work in the local development environment when USE_SSL is set to false. On the other hand, keeping http will work on both cases regardless of whether USE_SSL is set to true or false with the potential security implication ((although unlikely) as @ironfroggy noted above. So what is the best practice we should go with in this scenario? How github and others handle this when verifying user accounts? Do they use http or https? If using https is the best practice we should go with, I guess we just have to enforce USE_SSL to be true (or not even make it an option, just to set to use SSL by default) in our local development environment with a self-generated certificate.